geopandas/geoseries.py

```diff
@@ -199,12 +199,14 @@ def copy(self, order='C'):
         """
         # FIXME: this will likely be unnecessary in pandas >= 0.13
         return GeoSeries(self._geometry_array,  # TODO: implement copy
-                         index=self.index,
+                         index=self.index, crs=self.crs,
                          name=self.name).__finalize__(self)
 
     def apply(self, func, *args, **kwargs):
         s = Series(list(self._geometry_array), index=self.index)
-        return s.apply(func, *args, **kwargs)
+        L = s.apply(func, *args, **kwargs).tolist()
+        vec = from_shapely(L)
+        return GeoSeries(vec, index=self.index)
```

Comment:
I think the last two lines here are not needed, as typically `apply` will return a plain Series rather than a GeoSeries (in general, we could in principle detect the case when `L` holds all shapely objects)